packages/bolt-interactions/README.md

```diff
@@ -174,6 +175,36 @@ interactionFlow<FlowState>((flow, app) => {
 
 Create a [flow listener](#flow-listener) that listens for [actions](https://slack.dev/bolt/concepts#action-listening)
 
+## Working with Ids
+
+If you are working with your own ids for flow instances, it can be helpful to parse them. There some utilities to help with that.
```

Comment:
```suggestion
If you are working with your own ids for flow instances, it can be helpful to parse them. We provide some utilities to help with that.
```